config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-presets.yaml

```diff
@@ -27,3 +27,11 @@ presets:
       secretKeyRef:
         name: azure-secrets-store-cred
         key: tenantid
+- labels:
+    preset-gcp-secrets-store-creds: "true"
+  env:
+  - name: GCP_SA_JSON
+    valueFrom:
+      secretKeyRef:
+        name: gcp-secrets-store-cred
+        key: key.json
```

Comment:
Is this file already loaded in the prow cluster?

Reply:
this does _not_ exist in the prow cluster yet. starting that investigation now.

Reply:
The secret should exist now